dist/js/bootstrap-datepicker.js

```diff
@@ -6,11 +6,11 @@
 
 (function(factory){
     if (typeof define === 'function' && define.amd) {
-        define(['jquery'], factory);
+      define(['jquery'], factory(jQuery || window.jQuery || require('jquery')));
```

Comment:
Hey @ocarreterom , thanks for your comment but the UMD pattern is still in tact, the only change is to pass through variable declarations that can be different per environment. Happy to discuss if you have further feedback.

Reply:
@ocarreterom I´m using this plugin in an AMD environment and it fails with the error message described in the bugs description without this line.

Reply:
Did you test this line?
The second argument of `define([dependecies], callback)` should be a function (a callback like `factory`) to get the dependencies as params, not the result of the function `factory()`.

https://requirejs.org/docs/api.html#defdep

Reply:
Hi @ocarreterom, I did within a custom application but I´ll decline this PR until I´ve made sure I rule out any other reason why I experienced this error. Thanks for your patience.

Reply:
@ocarreterom sorry I can`t decline this PR nor can I give it a label. In case you have more access permissions could you please put a label "on hold" on it? Many thanks in advance.

Reply:
cc @vsn4ik